xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,1210 @@
+     </step>
+     <step>
+      <para>
+       Pull the docker registry container image from the &suse; registry.
+      </para>
+<screen>&prompt.user;<command>docker pull registry.suse.com/sles12/registry:2.6.2</command>
+   </screen>
+     </step>
+     <step>
+      <para>
+       Save the pulled image to a <literal>.tar</literal> file.
+      </para>
+<screen>&prompt.user;<command>docker save -o /tmp/registry.tar registry.suse.com/sles12/registry:2.6.2</command>
+      </screen>
+     </step>
+     <step>
+      <para>
+       Connect the trusted storage to the external mirror. Copy the
+       registry image onto the storage.
+      </para>
+<screen>&prompt.user;<command>cp <replaceable>/tmp/registry.tar</replaceable> <replaceable>/mnt/storage/registry.tar</replaceable></command>
+   </screen>
+     </step>
+     <step>
+      <para>
+       Create basic authentication credentials for the Docker registry.
+      </para>
+      <para>
+       Replace <literal>USERNAME</literal> and <literal>PASSWORD</literal> with
+       proper credentials of your choosing.
+      </para>
+<screen>&prompt.user;<command>mkdir auth</command>
+&prompt.user;<command>docker run --entrypoint htpasswd sles12/registry:2.6.2 -Bbn <replaceable>USERNAME PASSWORD</replaceable> > auth/htpasswd</command>
+ </screen>
+     </step>
+     <step>
+      <para>
+       Create the <filename>/etc/docker/registry/config.secure.yml</filename>
+       configuration file for Docker registry.
+      </para>
+      <para>
+       Replace <filename>/certs/domain.crt</filename> and
+       <filename>/certs/domain.key</filename> with the appropriate certificate
+       files for your organization.
+      </para>
+<screen>
+version: 0.1
+log:
+  fields:
+    service: registry
+auth:
+  htpasswd:
+    realm: basic-realm
+    path: /auth/htpasswd
+storage:
+  cache:
+    blobdescriptor: inmemory
+  filesystem:
+    rootdirectory: /var/lib/registry
+http:
+  addr: 0.0.0.0:5000
+  headers:
+    X-Content-Type-Options: [nosniff]
+  tls:
+    certificate: /certs/domain.crt
+    key: /certs/domain.key
+health:
+  storagedriver:
+    enabled: true
+    interval: 10s
+threshold: 3
+</screen>
+      <para>
+       For more details on the configuration, refer to:
+       <link xlink:href="https://docs.docker.com/registry/configuration/">Docker
+       Registry: Configuration</link>
+      </para>
+     </step>
+     <step>
+      <para>
+       Start the docker service and enable it at boot time:
+      </para>
+<screen>&prompt.root;<command>systemctl enable --now docker.service</command>
+         </screen>
+     </step>
+     <step>
+      <para>
+       Start the registry container.
+      </para>
+<screen>&prompt.user;<command>cd /etc/docker/registry</command>
+&prompt.user;<command>docker run -d -p 5000:5000 --restart=always --name registry \
+-v `pwd`<replaceable>/config.secure.yml</replaceable>:/etc/docker/registry/config.yml \
+-v `pwd`/auth:/auth \
+-v `pwd`/certs:/certs \
```

Comment:
We are pro users (:smile:), let's mark all the volumes read-only unless we really have to write to them.

```
-v `pwd`<replaceable>/config.secure.yml</replaceable>:/etc/docker/registry/config.yml:ro \
-v `pwd`/auth:/auth:ro \
-v `pwd`/certs:/certs:ro \
```